### PR TITLE
Fix constraint on protobuf version in gz-msgs python package

### PR DESCRIPTION
**Pull request by traversaro**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by traversaro** [comment]
@conda-forge-admin, please rerender

**Comment by traversaro** [comment]
@conda-forge-admin, please rerender

File: recipe/build_cxx.sh [hunk 1]
```diff
@@ -5,6 +5,8 @@ if [[ "${target_platform}" == osx-* ]]; then
     CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
 fi
 
+env
+
```
Comment (h-vetinari) [inline recipe/build_cxx.sh:9]: Is this a debug left-over?

**Review by h-vetinari — APPROVED** [review]
This should be ready, PTAL @conda-forge/gz-msgs!

It happens to be blocking the current protobuf migration

**Review by Tobias-Fischer — APPROVED** [review]

**Comment by Tobias-Fischer** [comment]
Happy to merge if @traversaro confirms it's all ready.

**Comment by traversaro** [comment]
Thanks for the feedback!

**Pull request merged**